vendor/cran.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
What about suffixing the files with `-test` (eg. `cran-test.js`)? That way, if/when we split services out into their own separate files, we'd have `cran.js` sitting alongside `cran-test.js`.